nixos/modules/programs/kdeconnect.nix

```diff
@@ -0,0 +1,28 @@
+{ config, pkgs,lib, ... }:
+with lib;
+{
+  options.programs.kdeconnect = {
+    enable = mkEnableOption ''
+      kdeconnect.Note that it will open the TCP and UDP port from
+      1714 to 1764 as they are needed for its correct working
+    '';
+    package = mkOption {
+      default = pkgs.kdeconnect;
+      defaultText = "pkgs.kdeconnect";
+      type = types.package;
+      description=''
+        The package providing the implementation for kdeconnect.
+      '';
+    };
+  };
+  config=
+    let
+      cfg = config.programs.kdeconnect;
+    in mkIf cfg.enable {
+      environment.systemPackages = [cfg.package];
```

Comment:
Perhaps also add gsconnect to systemPackages, if desktop is gnome 3?

Reply:
Maybe in a further commit